app/Modules/Bookings/BookingsService.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Modules\Bookings;
+
+use App\Constants\ControllerRating;
+use App\Exceptions\OverlappingBookingException;
+use App\Position;
+use App\User;
+use Carbon\Carbon;
+
+class BookingsService implements BookingsServiceInterface
+{
+    /** @var mixed App\User */
+    protected $user;
+
+    public function __construct(User $user)
+    {
+        $this->user = $user;
+    }
+
+    /**
+     * Check if the the user is able to book the given position only on the rating requirement.
+     *
+     * @param  User  $user
+     * @param  Position  $position
+     * @return bool
+     */
+    public function validateRatingRequirement(User $user, Position $position): bool
+    {
+        $positionSuffix = $position->suffix;
+
+        $ratingValue = $user->atcRating->vatsim_id;
+
+        return ControllerRating::isValidRatingForSuffix($positionSuffix, $ratingValue);
+    }
+
+    public function validateBookingTimes(Carbon $from, Carbon $to, Position $position): bool
```

Comment:
Still need to add the logic in here to exclude a certain booking from the search (i.e. the existing booking when updating)

Reply:
Done.